packages/fxa-auth-server/lib/senders/emails/partials/appBadges.ts

```diff
@@ -0,0 +1,21 @@
+export const appBadges = `<mj-section>
+    <mj-group>
+        <mj-column>
+            <mj-image css-class="app-badges" href="{{{iosUrl}}}" src="https://image.e.mozilla.org/lib/fe9915707361037e75/m/5/ebdfb903-4e62-4aeb-9eae-26e1ef495049.png" alt="ios-badge"></mj-image>
```

Comment:
Oh wow, I'm surprised we hardcode this URL in the old templates but we certainly do!

We may want to pass in "iOS badge" for the `alt` text as well as adjust the one for android, a screenreader may not read that differently than "ios-badge" but it _may_ and having a dash there seems odd.